src/main/java/seedu/address/logic/commands/EditCommand.java

```diff
@@ -146,7 +146,7 @@ private static Person createEditedPerson(Person personToEdit, EditPersonDescript
 
         } else if (personToEdit instanceof Runner) {
 
-            List<Customer> customers = editPersonDescriptor.getCustomers().orElse(((Runner) personToEdit)
+            List<Person> customers = editPersonDescriptor.getCustomers().orElse(((Runner) personToEdit)
```

Comment:
Just wondering, why `List<Person>` instead of a `List<Customer>`?

Reply:
Hi, hard to go into too much details here but it's because in the current implementation, when loading individuals from an XML file, XmlAdaptedPersons are created, then a method XmlAdaptedPersons.toModelType(), constructs Person, Customer and Runner objects to a Person reference instead of a specialized Customer reference. So the program can only compile when the looser definition, List<Person>, is used.

Reply:
It's possible I can create a new class XmlAdaptedCustomer extends XmlAdaptedPerson to address this issue, but I'm not sure if it will work. But anyway for now, the implementation in this PR is functional!

Reply:
Alright, great explanation, I understand!

Reply:
Great work!